src/ServiceControl/MessageFailures/FailedMessage.cs

```diff
@@ -26,6 +26,11 @@ public static string MakeDocumentId(string messageUniqueId)
             return $"{CollectionName}/{messageUniqueId}";
         }
 
+        public static string GetDocumentIdFromCollectionId(string collectionId)
```

Comment:
Is collection id the correct term? Wouldn't the better term be something like document id?

Reply:
Yep, you're correct

Reply:
Renamed